.github/workflows/php.yml

```diff
@@ -10,28 +10,35 @@ jobs:
   build-test:
 
     runs-on: ubuntu-latest
+    name: Run tests (${{ matrix.php }})
+
+    strategy:
+      matrix:
+        php: [8.1, 8.2]
```

Comment:
I refactored a little bit this file to be able to run the test suite on 2 versions of PHP.

(Please approve running the test suite! 🙏 )

Reply:
Nicely done.